client/components/account-balances/balance-tooltip.tsx

```diff
@@ -3,29 +3,165 @@
  */
 import React from 'react';
 import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import interpolateComponents from '@automattic/interpolate-components';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { ClickTooltip } from 'components/tooltip';
+import { documentationUrls, fundLabelStrings } from './strings';
+import InlineNotice from 'components/inline-notice';
 
 type BalanceTooltipProps = {
 	label: string;
 	content: React.ReactNode;
+	maxWidth?: string | undefined;
+};
+
+type TotalBalanceTooltipProps = {
+	balance: number;
+};
+
+type AvailableBalanceTooltipProps = {
+	balance: number;
 };
 
 const BalanceTooltip: React.FC< BalanceTooltipProps > = ( {
 	label,
 	content,
+	maxWidth,
 } ) => {
 	return (
 		<ClickTooltip
 			content={ content }
 			className="wcpay-account-balances__balances__item__tooltip"
 			buttonIcon={ <HelpOutlineIcon /> }
 			buttonLabel={ label }
+			maxWidth={ maxWidth }
 		/>
 	);
 };
 
-export default BalanceTooltip;
+export const TotalBalanceTooltip: React.FC< TotalBalanceTooltipProps > = ( {
+	balance,
+} ) => {
+	return (
+		<BalanceTooltip
+			label={ `${ fundLabelStrings.total } tooltip` }
+			maxWidth={ '315px' }
```

Comment:
Is this maxWidth to ensure the equation (`Total balance = Available funds + Pending funds`) is not wrapped? Would be good to note that in the code, so future readers know why and can iterate on how we handle that constraint.

Reply:
> I actually prefer for more simple diff if we are OK with narrower tooltip like that.

Hmm, I guess we could render the equation on two lines like this:

```
Total balance = 
Available funds + Pending funds
```

Not sure if that's clear enough, curious for thoughts from @rogermattic (and on other ideas below).


> Adding maxWidth was necessary because the [BalanceTooltip](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/account-balances/balance-tooltip.tsx#L18) renders [ClickTooltip](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/tooltip/index.tsx#L109) which renders [TooltipBase](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/tooltip/tooltip-base.tsx#L174) which [sets an inline max-width](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/tooltip/tooltip-base.tsx#L213) in a function that calculates tooltip position, window dimension etc, so it's pretty complex stuffs to modify.

I see! That's frustrating – an example of where an earlier decision / detail limits our options. 

- Can we undo this ? In your opinion, does `TooltipBase` need to force a max width in inline style, could it move to css?
- Can we make the base tooltip width bigger (i.e. make all tooltips wider) to work better for our tooltip?
- If `TooltipBase` needs to force a width, could it use `minWidth`, allowing overriding to make things wider?

Reply:
IMO, it looks good in one line.

> Would be good to note that in the code, so future readers know why and can iterate on how we handle that constraint.

Good call to put some description in a comment. Added comment in https://github.com/Automattic/woocommerce-payments/pull/8140/commits/d607985a091738d96aece19cf3a35e31ef985d06.

> In your opinion, does TooltipBase need to force a max width in inline style, could it move to css?

Can't tell for sure. I see that it's set and used in a `useEffect()` ([ref](https://github.com/Automattic/woocommerce-payments/blob/530783fe131414f5189259aed5dc862ad89f7d74/client/components/tooltip/tooltip-base.tsx#L256)).

> If TooltipBase needs to force a width, could it use minWidth, allowing overriding to make things wider?

IMO the current intention is to not let the tooltip content 'stretches' the tooltip without control. This is more important in narrow screen I think.

Changing `TooltipBase` needs to be done carefully as it might affect other UI and out of scope of this PR. Created https://github.com/Automattic/woocommerce-payments/issues/8269.

Reply:
> Hmm, I guess we could render the equation on two lines like this:
> 
> ```
> Total balance = 
> Available funds + Pending funds
> ```
> 
> Not sure if that's clear enough, curious for thoughts from @rogermattic (and on other ideas below).
> 

I prefer it when it renders in a single line, BUT I don't think it is necessary as it reads OK when split into two lines.